dataactcore/migrations/versions/3a79d7a4d1c1_merge_rule_tables.py

```diff
@@ -69,15 +69,15 @@ def upgrade_validation():
     op.drop_table('multi_field_rule_type')
     op.add_column('rule', sa.Column('file_id', sa.Integer(), nullable=True))
     op.add_column('rule', sa.Column('target_file_id', sa.Integer(), nullable=True))
-    op.create_foreign_key(None, 'rule', 'file_type', ['file_id'], ['file_id'])
+    op.create_foreign_key('fk_file_id', 'rule', 'file_type', ['file_id'], ['file_id'])
     op.create_foreign_key('fk_target_file_id', 'rule', 'file_type', ['target_file_id'], ['file_id'])
     ### end Alembic commands ###
 
 
 def downgrade_validation():
     ### commands auto generated by Alembic - please adjust! ###
     op.drop_constraint('fk_target_file_id', 'rule', type_='foreignkey')
-    op.drop_constraint(None, 'rule', type_='foreignkey')
+    #op.drop_constraint('fk_file_id', 'rule', type_='foreignkey')
```

Comment:
Thanks, this was a temporary thing due to naming this constraint while my DB was on a later version.  Uncommented it now.